src/ui/shared/BrutalistUI.jsx

```
@@ -0,0 +1,900 @@
+      }, Math.random() * 400 + 200); // Random typing delay
+      return () => clearTimeout(timer);
+    }
+  }, [currentIndex]);
+
+  return (
+    <div className="w-full h-48 border border-(--toxic-green)/30 bg-[color:var(--shadow-black)] p-4 font-mono text-xs overflow-y-auto flex flex-col gap-1 custom-scrollbar relative shadow-[inset_0_0_20px_var(--void-black)]">
+      {/* Scanline overlay */}
+      <div className="absolute inset-0 pointer-events-none opacity-10 bg-[linear-gradient(transparent_50%,var(--toxic-green-20)_50%)] bg-[length:100%_4px]"></div>
+
+      {lines.map((line, i) => (
+        <div key={i} className={`${line.includes('[ERROR]') ? 'text-[color:var(--blood-red)] font-bold' : line.includes('[WARN]') ? 'text-[color:var(--warning-yellow)]' : 'text-(--toxic-green)'} opacity-90 leading-relaxed`}>
```

Biome (2.4.4)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)